src/pages/material/material-add/material-add.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import MaterialForm from '../../../components/material-form';
+import { useStyles } from './material-add.styles';
+
+const MaterialAdd = () => {
```

Comment:
It is a useless component because it doesn't add any meaningful logic or jsx markup to the application

Reply:
it adds necessary styles. Also it is a way to use one common material-form for adding and editing. as example I used news.